apps/database-new/app/[threadId]/[runId]/page.tsx

```diff
@@ -1,33 +1,7 @@
-'use client'
-
-import { last, sortBy } from 'lodash'
-import { useParams, useRouter } from 'next/navigation'
-import { useEffect, useMemo } from 'react'
-
-import { useMessagesQuery } from '@/data/messages-query'
-
-export default function ThreadPage() {
-  const router = useRouter()
-  const { threadId, runId }: { threadId: string; runId: string } = useParams()
-
-  const { data, isSuccess } = useMessagesQuery({ threadId, runId })
-
-  const messages = useMemo(() => {
-    if (isSuccess) return sortBy(data.messages, (m) => m.created_at)
-    return []
-  }, [data?.messages, isSuccess])
-
-  useEffect(() => {
-    if (isSuccess && messages.length > 0) {
-      const latestMessage = last(messages.filter((message) => message.role === 'user'))
-      if (latestMessage) router.push(`/${threadId}/${runId}/${latestMessage.id}`)
-    }
-  }, [isSuccess])
-
+export default function ThreadPage({ children }: { children: React.ReactNode }) {
   return (
     <div className="grow max-h-screen flex flex-row items-center justify-between bg-alternative h-full">
-      {/* <SchemaGraph tables={[]} />
-      <CodeEditor content="" /> */}
+      {children}
```

Comment:
this is failing 
https://vercel.com/supabase/database-new/4WqED8ot96tviWLBUcsPNx4xQGM8

b/c pages don't accept children
https://stackoverflow.com/questions/76404674/type-error-page-has-an-invalid-default-export-type-is-not-valid

Reply:
ha, oh yea. thanks